sdk/core/Azure.Core/tests/public/PatchModels/ChildPatchModel.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core.Json;
+
+namespace Azure.Core.Tests.PatchModels
+{
+    /// <summary>
+    /// This model illustrates a nested child model in a parent model.
+    /// </summary>
+    public partial class ChildPatchModel
```

Comment:
We have a base test class [ModelJsonTests](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/tests/public/ModelSerializationTests/ModelJsonTests.cs) which run a model through all the serialization scenarios without you needing to code all of them.  Can we add a `ChildPatchModelTests : ModelJsonTests<ChildPatchModel>` test class to run at least one of these through those checks to validating we don't have any oddities.